pkgs/by-name/ki/kikoplay/qhttpengine.nix

```diff
@@ -0,0 +1,31 @@
+{
+  fetchFromGitHub,
+  stdenv,
+  lib,
+  cmake,
+  qt5,
+}:
+stdenv.mkDerivation rec {
```

Comment:
This could be a real package in pkgs/by-name, I think.

Reply:
The `qhttpengine` package hasn't had an update over the last 6 years. It should not be used in any other package than KikoPlay.